docs/sources/writing-guide/reuse-shared-content.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Reuse shared content
+description: How to reuse chunks of content between pages
+keywords:
+  - content reuse
+  - shared content
+---
+
+# Reuse shared content
+
+Shared content is a sentence, paragraph, or whole page of text that makes sense in multiple contexts.
+Reusing shared content propagates changes from a single source file to one or more destination pages.
+
+This topic describes how to extract and share a chunk of content to multiple pages.
+
+## Before you begin
+
+- Identify a chunk of content that should be reused in multiple pages.
+
+## Steps
+
+To reuse shared content:
+
+1. Identify the sharing project.
+
+   If the shared content is to be reused in a single project, then the sharing project is that project.
```

Comment:
my brain did somersaults trying to parse this. had to read it a couple of times to fully understand it. I recommend adding an example of both use cases, which will help to clarify these words.

Reply:
Tried rewording in https://github.com/grafana/writers-toolkit/pull/189/commits/addd9f9a389d73653cf6634412f2b0a4fecb8871